templates/alipay-mini-game/game.ejs

```diff
@@ -44,13 +41,7 @@ function onApplicationCreated(application) {
         // Adjust devicePixelRatio
         cc.view._maxPixelRatio = 5;
         // Release Image objects after uploaded gl texture
-        cc.macro.CLEANUP_IMAGE_CACHE = false;
```

Comment:
we don't use `cc.view._maxPixelRatio` anymore, can be removed now

Reply:
why use that before

Reply:
it's a little bit hack bofore, to reduce the dpr like we do on WEB platforms